src/common/tableTools/applicationTableTools/ApplicationTableTools.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { TextInput } from 'hds-react';
+
+import { ApplicationStatus } from '../../../@types/__generated__/globalTypes';
+import Select from '../../select/Select';
+import { APPLICATION_STATUS } from '../../utils/constants';
+import styles from './applicationTableTools.module.scss';
+
+interface ApplicationStateTableToolsProps {
```

Comment:
This needs to be renamed

Reply:
Yes, looking into how to combine the two new table tools recently introduced for the next PR.